src/NUnitEngine/Addins/nunit-v2-result-writer/NUnit2ResultSummary.cs

```diff
@@ -101,7 +100,6 @@ private void Summarize(XmlNode node)
                         case "Ignored":
                             ignoreCount++;
                             break;
-                        case "Skipped":
```

Comment:
Usually "redundant code" == "the compiler doesn't need it to understand"
Coding standards are about what the reader needs